packages/core/admin/admin/src/content-manager/components/ListViewTable/Body/index.js

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link, useHistory } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+
+import { BaseCheckbox, IconButton, Tbody, Td, Tr, Flex } from '@strapi/design-system';
+import { Trash, Duplicate, Pencil } from '@strapi/icons';
+import { useTracking, stopPropagation, onRowClick, useTableContext } from '@strapi/helper-plugin';
+
+import { usePluginsQueryParams } from '../../../hooks';
+import ConfirmDialogDelete from '../ConfirmDialogDelete';
+
+/* -------------------------------------------------------------------------------------------------
+ * DataCell
+ * -----------------------------------------------------------------------------------------------*/
+
+const DataCell = ({ children, ...props }) => {
+  return <Td {...props}>{children}</Td>;
+};
+
+DataCell.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * CheckboxDataCell
+ * -----------------------------------------------------------------------------------------------*/
+
+const CheckboxDataCell = ({ rowId, index }) => {
+  const { selectedEntries } = useTableContext();
+  const { formatMessage } = useIntl();
+  const isChecked = selectedEntries.findIndex((id) => id === rowId) !== -1;
+  const { onSelectRow } = useTableContext();
+  const ariaLabel = formatMessage(
+    {
+      id: 'app.component.table.select.one-entry',
+      defaultMessage: `Select {target}`,
+    },
+    { target: index + 1 }
+  );
+
+  return (
+    <DataCell {...stopPropagation}>
```

Comment:
I'd love to have your opinion on this – I hate the `stopPropagation` util, this is the code:

```js
export const stopPropagation = {
  onClick: (e) => e.stopPropagation(),
  role: 'button',
  'aria-hidden': true,
};
```

This therefore changes the `DataCell` to a button and hides EVERYTHING in it, which i think on both accounts is very wrong, the click handler makes sense but the role and `aria-hidden` not for me – what do you think?